index.html

```diff
@@ -35,11 +35,11 @@
         <div class="body-class">
                 
                                 <ul class="social_icon">
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/gi.svg"></a></li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/in.svg"></a> </li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Enabled.svg"></a></li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Vector.svg"></a></li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Disabled.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Github icon" src="imgs/gi.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Linkedin Icon" src="imgs/in.svg"></a> </li>
+                                        <li><a href="#"><img class="icon" alt="Social media Icon" src="imgs/Enabled.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="social media icon" src="imgs/Vector.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Social media Icon" src="imgs/Disabled.svg"></a></li>
                                   </ul>
                  
 
```

Comment:
- [ ] In line 70 you have another h1 tag which is completely against the [hierarchy of headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). Also in many places, you didn't follow the [hierarchy of headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). Please follow the hierarchy :+1: